src/Info.elm

```diff
@@ -123,8 +123,8 @@ info =
       , slug = "about"
       , icon = "question-circle-o"
       , infoText =
-            [ """In 2017, The Haven, Wolverhampton, Neontribe and women who had been through domestic abuse co-designed a web app to encourage other women in
-            that situation to seek support and advice. We want to find out if the structure of that app could be used by different organisations. We believe
+            [ """In 2017, The Haven, Wolverhampton worked with Neontribe and women who had been through domestic abuse. We co-designed a web app to encourage other women in that situation to seek support and advice.
```

Comment:
The Haven Wolverhampton is all one thing. No comma on their own website.

Reply:
Should it be The Haven Wolverhampton,  with a comma or without?

Reply:
I think `The Haven Wolverhampton worked with Neontribe...`